### PR TITLE
Bug 1588266 Update definition for openshift_logging_es_pvc_dynamic variable

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1588266

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@bmcelvee - hey this was verified on 9/July by Anping. Can this be moved along please?

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -427,15 +427,18 @@ must be at least 512M. Possible suffixes are G,g,M,m.
 |A key/value map added to a PVC in order to select specific PVs.
 
 |`openshift_logging_es_pvc_dynamic`
-|Set to `true` to have PVC claims annotated so that their backing storage is
-dynamically provisioned. When set to `true` the storage class is ignored and not
-set to the PVC.
+|Set to `true` so the backing storage is dynamically provisioned. When set to
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/Set to `true` so the backing storage is dynamically provisioned./ To dynamically provision the backing storage, set the parameter value to \`true`.

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -427,15 +427,18 @@ must be at least 512M. Possible suffixes are G,g,M,m.
...
+`true` the storageClass spec is omitted from the PVC definition. The setting of
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/`true`/`true`,

For the last sentence, do you mean "If you set a \`openshift_logging_es_pvc_storage_class_name\` parameter value, its value overrides the value of the the \`openshift_logging_es_pvc_dynamic\` parameter."?

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -427,15 +427,18 @@ must be at least 512M. Possible suffixes are G,g,M,m.
...
+`openshift_logging_es_pvc_storage_class_name` can override
+`openshift_logging_es_pvc_dynamic`.
 
 |`openshift_logging_es_pvc_storage_class_name`
 |To use a non-default storage class, set the variable with the storage class
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/set the variable with/specify

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -427,15 +427,18 @@ must be at least 512M. Possible suffixes are G,g,M,m.
...
 name. For example, set to one of the following,
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: I'd say, "name, such as \`glusterprovisioner\` or \`cephrbdprovisioner\`. After you specify the storage class name, dynamic volume provisioning is active regardless of the  `openshift_logging_es_pvc_dynamic` value."

File: install_config/aggregate_logging.adoc [hunk 5]
```diff
@@ -992,7 +995,7 @@ $ oc patch secrets/logging-fluentd --type=json \
 
 [NOTE]
 ====
-Replace `your_private_key` with a generic name. This is a link to the JSON path, 
+Replace `your_private_key` with a generic name. This is a link to the JSON path,
 not a path on your host system
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/system/system.

File: install_config/aggregate_logging.adoc (hunk 2 above)
Comment (bmcelvee) [reply install_config/aggregate_logging.adoc]: Yes, thanks!

**Comment by bmcelvee** [comment]
Thanks so much, @kalexand-rh! :cat2:

**Pull request merged**